components/root/root.jsx

```diff
@@ -187,13 +187,13 @@ export default class Root extends React.Component {
         /*eslint-enable */
 
         const afterIntl = () => {
-            initializePlugins();
-
             if (this.props.location.pathname === '/' && this.props.noAccounts) {
                 this.props.history.push('/signup_user_complete');
             }
 
-            this.setState({configLoaded: true});
+            initializePlugins().then(() => {
```

Comment:
Just so I understand, is this just cleanup and not directly relevant, or is this so that we now wait until all translations are loaded inside initializePlugin()?

Reply:
We now wait for all plugins to be loaded.